archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/ClassesThat.java

```diff
@@ -391,6 +394,9 @@
     /**
      * Matches classes that implement a certain interface with the given type name. This is equivalent to
      * {@link #implement(Class)}, but does not depend on having a certain type on the classpath.
+     * Note that this only matches non-interface {@link JavaClass classes} that implement the supplied interface {@code typeName}
+     * (compare {@link JavaClass.Predicates#implement(Class)}.
```

Comment:
```suggestion
     * (compare {@link JavaClass.Predicates#implement(String)}.
```

Reply:
Hmm, this was on purpose, because only `implement(Class)` has the explanation :thinking: If you jump to the `String` version you'll just find another link to the `Class` version :man_shrugging: